Exiled.API/Features/Player.cs

```diff
@@ -1528,6 +1528,21 @@ public byte GetEffectIntensity<T>()
             throw new ArgumentException("The given type is invalid.");
         }
 
+        /// <summary>
+        /// Gets all the active <see cref="PlayerEffect"/>.
+        /// </summary>
+        public List<PlayerEffect> GetAllEffects()
+        {
+            List<PlayerEffect> ActiveEffects = new List<PlayerEffect>();
+            for (int i = 0; i < (int) EffectType.Visuals939; i++)
```

Comment:
`for (int i = 0; i < Enum.GetValue(typeof(EffectType)).GetUpperBound(0); i++)` so we don't have to update this everytime a new effect type is added.